app/models/user.rb

```diff
@@ -114,13 +114,9 @@ def active_for_authentication?
   end
 
   def inactive_message
-    if active? && enabled?
-      super
-    elsif softly_deleted?
-      'This account has been deleted.'
-    elsif disabled?
-      'This account has been disabled.'
-    end
+    return 'Invalid Email or password.' unless active? && enabled?
+
```

Comment:
Does this mean, even the a user has been disabled / deleted they will see the same message as 'Invalid Email or Password' ?

Reply:
Yes, for security reasons if we keep the account has been deleted/disabled, these messages are just saying the details you have put in is correct however the account has been deleted/disabled which we don't want if the user trying to login is not the actual user.